website/docs/r/org_ldap.html.markdown

```diff
@@ -115,25 +119,25 @@ The `custom_settings` section contains the configuration for the LDAP server
 <a id="user-attributes"></a>
 ### User Attributes
 
-* `object_class` - (Required)  LDAP _objectClass_ of which imported users are members. For example, _user_ or _person_
-* `unique_identifier` - (Required) LDAP attribute to use as the unique identifier for a user. For example, _objectGuid_
-* `username` - (Required) LDAP attribute to use when looking up a username to import. For example, _userPrincipalName_ or _samAccountName_
-* `email` - (Required) LDAP attribute to use for the user's email address. For example, _mail_
-* `display_name` - (Required) LDAP attribute to use for the user's full name. For example, _displayName_
-* `given_name` - (Required) LDAP attribute to use for the user's given name. For example, _givenName_
-* `surname` - (Required) LDAP attribute to use for the user's surname. For example, _sn_
-* `telephone` - (Required) LDAP attribute to use for the user's telephone number. For example, _telephoneNumber_
-* `group_membership_identifier` - (Required) LDAP attribute that identifies a user as a member of a group. For example, _dn_
+* `object_class` - (Required)  LDAP _objectClass_ of which imported users are members. For example, _"user"_ or _"person"_ attributes
+* `unique_identifier` - (Required) LDAP attribute to use as the unique identifier for a user. For example, the _"objectGuid"_ attribute
+* `username` - (Required) LDAP attribute to use when looking up a username to import. For example, _"userPrincipalName"_ or _"samAccountName"_ attributes
+* `email` - (Required) LDAP attribute to use for the user's email address. For example, the _"mail"_ attribute
+* `display_name` - (Required) LDAP attribute to use for the user's full name. For example, the _"displayName"_ attribute
+* `given_name` - (Required) LDAP attribute to use for the user's given name. For example, the _"givenName"_ attribute
+* `surname` - (Required) LDAP attribute to use for the user's surname. For example, the _"sn"_ attribute
+* `telephone` - (Required) LDAP attribute to use for the user's telephone number. For example, the _"telephoneNumber"_ attribute
+* `group_membership_identifier` - (Required) LDAP attribute that identifies a user as a member of a group. For example, the _"dn"_ attribute
```

Comment:
This is a late comment about `group_back_link_identifier` param below. I just noticed that though docs say it's optional, it actually fails if you don't provide it:

```
│ Error: Missing required argument
│
│   on main.tf line 395, in resource "vcfa_org_ldap" "my-org-ldap":
│  395:     group_attributes {
│
│ The argument "group_membership_identifier" is required, but no definition was found.
```